basculechecks/capabilitiesvalidator.go

```diff
@@ -93,6 +94,9 @@ func (c CapabilitiesValidator) CheckAuthentication(auth bascule.Authentication,
 	if auth.Token == nil {
 		return MissingValues, ErrNoToken
 	}
+	if len(auth.Request.Method) == 0 {
```

Comment:
Can server-side requests ever have empty methods?

Reply:
If the `Authentication` is built incorrectly, it can.  We build it in the constructor middleware, but it's not technically required.  Anyone can build their own `Authentication` and put it into a context:
https://github.com/xmidt-org/bascule/blob/main/context.go